### PR TITLE
fix: update config for move to chinese subpath

### LINKED ISSUE #46197: Update 11ty news for Chinese
https://github.com/freeCodeCamp/news/blob/b2585c986c8627ca4312a8dde3fee5d0141d2fa1/config/index.js#L89

### PR DESCRIPTION
**Pull request by raisedadead**
Closes freeCodeCamp/freeCodeCamp#46197

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
I believe this should be ready to handle the move to the `/chinese` subpath. Once Ghost is moved over, all we should have to do is update the `CHINESE_GHOST_API_URL` environment variables on Orbit and Azure.

Moving the Ghost instance will probably break a lot of the hotlinked images for the posts in the Dockerized Chinese Ghost instance. But that's not a bit deal -- I can fix those in a future PR. The broken images shouldn't affect the current tests.

Edit: Added the blocked label until everything else is in place.

**Review by ojeytonwilliams — DISMISSED** [review]
For what it's worth, this LGTM

File: cypress/e2e/chinese/landing/landing.cy.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 describe('Landing', () => {
   before(() => {
+    // Update baseUrl to include current language
+    Cypress.config('baseUrl', 'http://localhost:8080/chinese/news');
```
Comment (ShaunSHamilton) [inline cypress/e2e/chinese/landing/landing.cy.js:4]: I assume to fix the tests, this also needs to be done in `ads.cy.js` for `/chinese`. However, I do not have enough context to be sure whether it makes sense considering ads in China are treated differently?

Comment (raisedadead) [reply cypress/e2e/chinese/landing/landing.cy.js:4]: We are not treating Chinese to be used exclusively in mainland China anymore. That is the aim of this migration is to cater to the Global Chinese-speaking audience.

**Pull request merged**